### DISCUSSION TITLE
One to many relationship between streams

### DISCUSSION THREAD

**Discussion by ShivaniSK** [finos/TimeBase-CE#254] [Q&A]
Hello @alex-karpovich 
Is there a way to create streams with one-to-many relationship?
And then query timebase by joining the streams?

**Comment by alex-karpovich** [comment]
Nope, TimeBase do not support relations, like classical SQL databases.

**Reply by ShivaniSK** [reply]
okay thanks @alex-karpovich , also is there a way to index another column apart from "symbol and timestampMs" which are the default ones?

**Reply by alex-karpovich** [reply]
Please describe your use case

**Reply by ShivaniSK** [reply]
Hello @alex-karpovich 
Normally in classic SQL, we can index any column so that when executing a query with "where clause" on the column , the performance is faster. E.g if we have a new column "ABC" in timebase stream "xyz" and i want to always execute queries like below
"select * from xyz where ABC = someValue", in classic SQL if we index this column ABC, then the performance improves.
Is there any similar concept of indexing in Timebase?

**Reply by alex-karpovich** [reply]
We are planning indexing support for next year, but for now we have good reading performance that helps to get good result with linear scan
